### ISSUE TITLE
toggle-notes, only toggles off

### ISSUE THREAD

**Issue by coltongrainger** [damianavila/RISE#434]
### expected result

To toggle notes with a user defined hotkey.

### observed

Defined hotkey only removes slide type. 

### MWE

minimal .json to reproduce

```
"rise": {
    "shortcuts": {
      "toggle-notes": "shift-n",
    }
}
```

Thanks!

**Comment by parmentelat** [comment]
Hi; indeed this is not working as advertised  
I'd like to take this chance to start a more general discussion on these keyboard shortcuts  
Several things IMO could are suboptimal at this point:  
* first off, the Shift-u (for toggle-subslides) shortcut somehow seems to be bound be something else to 'move up'; or it is in my environment only ?
* second, the doc (and nbextensions panel) wrongly advertises toggle-fragment as being bound to Shift-f while it seems to be Shift-g instead, at least in my environment; I remember having hesitated between for some reason, but I can't remember that reason now..
* third, as you pointed out, toggle-notes does does to behave as advertised

Finally, and more generally, I'm not even sure that the shortcuts themselves were picked in an optimal way. I decided to have them based on Shift instead of Alt, but I can't remember the rationale here again either.

So in other words, maybe now that this has been out for a while, maybe others would like to jump in and propose some alternate choice for the shortcuts. Otherwise I guess the issue would boil down to addressing the 3 items above

my $0.02

**Comment by damianavila** [comment]
> first off, the Shift-u (for toggle-subslides) shortcut somehow seems to be bound be something else to 'move up'; or it is in my environment only ?

Could be something from the native keyboard event, I need to check.

> second, the doc (and nbextensions panel) wrongly advertises toggle-fragment as being bound to Shift-f while it seems to be Shift-g instead, at least in my environment; I remember having hesitated between for some reason, but I can't remember that reason now..

Maybe you have seen some issues with `f`? Same problem as above?

> third, as you pointed out, toggle-notes does does to behave as advertised

We need to fix it :wink:

> Finally, and more generally, I'm not even sure that the shortcuts themselves were picked in an optimal way. I decided to have them based on Shift instead of Alt, but I can't remember the rationale here again either.

I think Shift is probably a good choice. We just need to check we are not overlapping other events.

> So in other words, maybe now that this has been out for a while, maybe others would like to jump in and propose some alternate choice for the shortcuts. Otherwise, I guess the issue would boil down to addressing the 3 items above

I am open to exploring other options. As you said, let's give some more time for feedback, otherwise, I think addressing the 3 items is enough for now.
